gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/jupiter/handlers/api/adapter/invoker/InvokerAdapter.java

```diff
@@ -80,6 +80,9 @@ public Completable invoke(RequestExecutionContext ctx) {
                     try {
                         // Invoke to make the connection happen.
                         legacyInvoker.invoke(adaptedCtx, streamAdapter, connectionHandlerAdapter);
+                        if (!nextEmitter.isDisposed()) {
```

Comment:
This should be done here and should be handled through connectionHandlerAdapter. Why did you need it ?